subt/main.py

```diff
@@ -938,23 +979,43 @@ def play_virtual_part_explore(self):
 
         self.stdout(self.time, "Explore phase finished %.3f" % dist, reason)
 
-    def play_virtual_part_map_and_explore_frontiers(self):
+    def play_virtual_part_map_and_explore_frontiers(self, details=None):
```

Comment:
What is the connection between "follow a drone" and "explore frontiers"? I would never look for a drone following functionality in a function named like this.

Reply:
definitely misleading name - original idea was just to feed the waypoints and the planner is outside of `main.py` - probably name like `play_virtual_part_follow_waypoints` would be better ... yeah with the hack to switch to left/right wall if it fails, hmm